action.yml

```diff
@@ -11,6 +11,8 @@ inputs:
   commitMessage: { "description": 'TODO', required: false }
   prTitle: { "description": 'TODO', required: false }
   prBody: { "description": 'TODO', required: false }
+  repo: { "description": 'TODO', required: false }
+  owner: { "description": 'TODO', required: false }
```

Comment:
This isn't used anywhere (yet)?

Reply:
They are propagated through via the action constructor, but i think i can get away with setting `GITHUB_REPOSITORY` so this might not be needed.  I'll remove it and see

Reply:
I tried passing this as part of the ENV but it was ignored

Reply:
Ah, I was going to say "you'll need to add them to https://github.com/timbertson/self-update-action/blob/f417b06edcafd811cfe9129727f36fb74d13ce60/src/lib.ts#L31" but now I see that I already listed them, just forgot to actually include in `action.yml`. Got it 👍